Added Lightrun and debugging/APM section (#10)
agamm/awesome-developer-first: README.md

*Storing data and processing it.*
* [Airtable](https://airtable.com/) - Database and spreadsheet hybrid with an API.

## Debugging 
*Tools that help in tracking bugs.*

> * [Google Cloud's operations suite](https://cloud.google.com/products/operations) -  Set of debug primitives (capture, log injection etc) for Google cloud

Reviewer: ```suggestion
* [Google Cloud's operations suite](https://cloud.google.com/products/operations) -  Set of debug primitives (capture, log injection, etc) for Google cloud.
```

Reviewer: Are you sure this is a widely known solution? I mean if people use this they don't need to use the other ones? plus it only seems to be relevant to GCP, no?

Author: The GCP option is pretty widely known. It's mostly known as stackdriver but Google renamed it recently.

> * [Lightrun](https://www.lightrun.com) - Debug production servers using a plugin in the IDE (on prem or SaaS) supports Java, Node.js and Python

Reviewer: ```suggestion
* [Lightrun](https://www.lightrun.com) - Debug production servers using a plugin in the IDE (on-prem or SaaS) that supports Java, Node.js, and Python.
```

> * [Rookout](https://www.rookout.com) - Debug production servers via cloud based debugger. Supports Java, .Net, Node.js, Python, and Ruby

Reviewer: ```suggestion
* [Rookout](https://www.rookout.com) - Debug production servers via a cloud-based debugger. Supporting Java, .Net, Node.js, Python, and Ruby.
```

Reviewer: Does rookout also support on-prem in some way? what is the other difference from lightrun? (to be consistent with the "on-prem or SaaS" you wrote :)

Author: Rookout supports more platforms and has been around longer. It's a cloud based solution so while it does have an IDE plugin it's main focus is on debugging via the web interface.  It doesn't have an on-prem option but it does have a locally installed proxy option. I'm not sure if that can be considered on-prem.

The three tools are conceptually similar. They try to provide the power of debugging local applications on a production server securely. Their approaches are also similar: agent installed on your server connected to their backend. The differences are:

- Agent implementations - this varies a bit but is too subjective to quantify so not relevant here
- Languages supported
- Vendor specific - Google only runs on GCP as far as I know
- IDE integration vs. Web App
- On-prem vs. SaaS - AFAIK only Lightun offers an on-prem option

Reviewer: Awesome, thanks for the clarification. Let's commit the grammar changes and merge?

Author: Sorry I thought I committed that, should be done now.
